src/xia2/cli/multiplex.py

```diff
@@ -204,6 +198,30 @@ def run(args=sys.argv[1:]):
                 f"symmetry.space_group has been set to: {params.symmetry.space_group}"
             )
 
+    if params.clustering.output_clusters and not params.reference:
+        logger.info(
+            "WARNING: clustering selected but no reference given. "
```

Comment:
Is this only an issue in the case of potential indexing ambiguity? Also I can't remember why this should occur anyway - is it because the clustering is performed on the scaled output - should we just be running on the reindexed file rather than scaled file and then this would solve this?

Reply:
I'm not 100% sure - I mostly just remember that without a reference I would get clusters output from a single job of multiplex that were in different settings. For example, if I had 10 clusters output from a single job, only say 4 of them would refine successfully (this was again with cubic insulin). I'm not sure if we ever got to the bottom of why it was inconsistent, but I can look into this a bit further.